beta-20161103.md

```diff
@@ -0,0 +1,72 @@
+---
+title: What's New in beta-20161103
+toc: false
+summary: Additions and changes in CockroachDB version beta-20161103.
+---
+
+## Oct 27, 2016
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161103.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20161103.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- Users (other than `root`) must now be created (with the `CREATE USER` statement or `cockroach user set` command) before being used. [#9794](https://github.com/cockroachdb/cockroach/pull/9794)
```

Comment:
You can't log in as the user until it has been created (this is obviously true for passwords since you have to give the user a password, but we're now enforcing it for certificates as well). We should probably also require the user to exist before it can be used in a GRANT statement, but we'll have to leave that for a future release.